CMSgov/mint-app
# [NOREF] Add closure to translation export

# NOREF


## Changes and Description

- Modified export to call closures for parent/child relation
- Updated unit test



## How to test this change

Verify that the export evaluates the closures to represent the parent/child obj configs

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
